_data/site_menu.yml

```diff
@@ -178,66 +161,99 @@ administration:
           - title: IBM Cloud Object Storage Providers
             path: "/docs/reference/latest/managing_providers/storage_providers/ibm_cloud_object_storage_managers.html"
             desc: "Managing IBM Cloud Object Storage Providers"
-
     - title: Provisioning Virtual Machines and Hosts
       path: "/docs/reference/latest/provisioning_virtual_machines_and_hosts/index.html"
       desc: "Provisioning, workload management, and orchestration in ManageIQ"
-
+    - title: Creating a Service for Virtual Machine Provisioning
+      path: "/docs/reference/latest/creating_a_service_for_vm_provisioning/index.html"
+      desc: "How to create a service for virtual machine provisioning"
+    - title: Service Provisioning Using an Orchestration Template
+      path: "/docs/reference/latest/service_provisioning_using_an_orchestration_template/index.html"
+      desc: "How to create an orchestration template to deploy services"
+    - title: Managing controls
+      children:
+        - title: Monitoring, Alerts, and Reporting
+          path: "/docs/reference/latest/monitoring_alerts_and_reporting/index.html"
+          desc: "Information on the Cloud Intelligence area of ManageIQ: dashboards, widgets, reporting, chargeback, optimization, and alerts"
+        - title: Policies and Profiles Guide
+          path: "/docs/reference/latest/policies_and_profiles_guide/index.html"
+          desc: Policy-based enforcement, compliance, events, and policy profiles for ManageIQ
+        - title: Chargeback
+          path: "/docs/reference/latest/monitoring_alerts_and_reporting/index.html#chargeback"
+          desc: Calculate monetary virtual machine charges based on owner or company tag
+    - title: Assigning a Custom Analysis Profile to a Virtual Machine
+      path: "/docs/reference/latest/assigning_custom_analysis_profile_to_vm/index.html"
+      desc: "How to assign custom analysis profile to a virtual machine"
     - title: Scripting Actions in ManageIQ
       path: "/docs/reference/latest/scripting_actions/index.html"
-      desc: "Real-time, bi-directional process integration for ManageIQ Management Engine"
+      desc: "Real-time, bi-directional process integration for ManageIQ"
```

Comment:
Ah, we got rid of Management Engine here.  We should do it throughout this page at the very least.  We can drop it elsewhere afterwards.